quiche/src/recovery/bbr/mod.rs

```diff
@@ -287,24 +287,22 @@ fn reset(r: &mut Recovery) {
     init::bbr_init(r);
 }
 
-fn on_packet_sent(r: &mut Recovery, sent_bytes: usize, _now: Instant) {
+fn on_packet_sent(r: &mut Recovery, _sent_bytes: usize, _now: Instant) {
     per_transmit::bbr_on_transmit(r);
-
-    r.bytes_in_flight += sent_bytes;
```

Comment:
This seems to change the behaviour of the code, rather than just being a refactoring?

`bbr_handle_restart_from_idle()` (called by `bbr_on_transmit()`) checks if `bytes_in_flight` is `0`, but if `bytes_in_fligh` is incremented before that call, then the condition will always be false after this change?

Reply:
Actually, disregard this, I misread the change.